_data/education.yml

```diff
@@ -34,6 +34,16 @@ websites:
       facebook: babbel.languages
       email_address: support@en.babbel.com
 
+    - name: California Institute of Technology
```

Comment:
How about adding the suffix (Caltech) at the end to aid the search feature on our site? I think most people would search for that instead of the full name.

Reply:
Good Idea. I'll add that in